config/core/src/test/groovy/org/apache/shiro/config/IniTest.groovy

```diff
@@ -119,24 +135,40 @@ public class IniTest {
         assertEquals("Truth", kv[0]);
         assertEquals("Beauty", kv[1]);
 
+        // Escape characters are to be removed from the key.
+        // This is different behaviour compared to the XML config.
```

Comment:
This seems odd, but I guess it's existing functionality

Reply:
Yes, I kept that. They just disappear magically.